### ISSUE TITLE
Errors during lisk-service docs site generation

### ISSUE THREAD

**Issue by TalhaMaliktz** [LiskHQ/lisk-service#1727]
### Actual behavior

```
    source: https://github.com/LiskHQ/lisk-service.git (branch: development | start path: docs/antora)
[14:46:47.801] ERROR (asciidoctor): target of xref not found: reference/configuration.adoc
    file: docs/antora/modules/ROOT/pages/configuration/docker.adoc
    source: https://github.com/LiskHQ/lisk-service.git (branch: development | start path: docs/antora)
[14:46:47.904] ERROR (asciidoctor): target of xref not found: reference/configuration.adoc
    file: docs/antora/modules/ROOT/pages/setup/source.adoc
    source: https://github.com/LiskHQ/lisk-service.git (branch: development | start path: docs/antora)

```
### Expected behavior
There shouldn't be such errors.
The correct path for the `configuration.adoc` is `configuration/index.adoc`

### Steps to reproduce
Execute `antora preview-site.yml --fetch` in lisk-docs repo

### Which version(s) does this affect? (Environment, OS, etc...)
lisk-service 0.7 docs

**Comment by chris529** [comment]
Please note: This issue has been fixed in PR #1722

**Comment by chris529** [comment]
Please note: This issue has now been fixed in PR #1722
